proposals/restructure-KF-docs-proposal.md

```diff
@@ -0,0 +1,228 @@
+This is an initial proposal, to be iterated on, following the discussion in [#2293](https://github.com/kubeflow/website/issues/2293).
+
+### Kubeflow documentation structure
+
+- **Getting Started**
+  - Introduction (current About/Kubeflow)
+  - Overview (current Kubeflow Overview)
+  - Use Cases (current About/Use Cases)
+  - Installing Kubeflow
+  - Cloud Installation
+    - AWS
+    - Azure
+    - Google Cloud
+    - IBM Cloud
+  - Kubernetes installation
+    - Overview of Deployment on Existing Clusters
+    - Kubeflow Deployment with kfctl_k8s_istio
+    - Multi-user, auth-enabled Kubeflow with kfctl_existing_arrikto
+    - Multi-user, auth-enabled Kubeflow with kfctl_istio_dex
+  - Workstation Installation
+    - Kubeflow on Linux
+    - Kubeflow on macOS
+    - Kubeflow on Windows
+    - MiniKF
+    - Deploy using MiniKF on GCP
+    - Deploying with minikube on a single node
+    - Kubeflow on MicroK8s
+    
+- **Community**
+  - About (current About/Community)
+  - Contributing (current About/Contributing to Kubeflow)
+  - Documentation (merge current About/Docs + About/Style Guide for Kubeflow docs)
+  - Events (current About/Events Calendar)
+  - Google Summer of Code (keep)
+  - Working Groups (new)
```

Comment:
Would it make sense to have a `Community/WGs` page?

Reply:
Perhaps this should live under `Contributing`?

Reply:
That makes sense. I agree that we should have a well structured `Contributing` page which may include this. I will make the change.